### PR TITLE
Font Size Bug Fix

### PR DESCRIPTION
**Pull request by jayfyle**
## Accessibility Fix: Font Size Change Bug
In case the user has increased/decreased the font size in the native OS settings, it would override the font styles defined in the app causing the UI to break.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -6,8 +6,8 @@ android {
         applicationId "com.ionicframework.fyle595781"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 51000
-        versionName "5.10.0"
+        versionCode 53300
```
Comment (yksolanki9) [inline android/app/build.gradle:9]: Revert changes in this file

File: android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <string name="package_name">com.ionicframework.fyle595781</string>
   <string name="custom_url_scheme">com.ionicframework.fyle595781</string>
   <string name="ionic_app_id">32316914</string>
-  <string name="ionic_channel_name">LIVE_UPDATE_CHANNEL</string>
+  <string name="ionic_channel_name">Staging</string>
```
Comment (yksolanki9) [inline android/app/src/main/res/values/strings.xml:8]: Revert this change

File: ios/App/App.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -371,12 +371,12 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = App/App.entitlements;
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 5.10.0;
+				CURRENT_PROJECT_VERSION = 5.33.0;
```
Comment (yksolanki9) [inline ios/App/App.xcodeproj/project.pbxproj:374]: Revert changes in this file

File: ios/App/App/Info.plist [hunk 4]
```diff
@@ -2,84 +2,100 @@
...
-    <key>CFBundleName</key>
-    <string>$(PRODUCT_NAME)</string>
-    <key>CFBundlePackageType</key>
-    <string>APPL</string>
-    <key>CFBundleShortVersionString</key>
-    <string>1.0</string>
-    <key>CFBundleURLTypes</key>
-    <array>
+      <key>CFBundleDevelopmentRegion</key>
+      <string>en</string>
```
Comment (yksolanki9) [inline ios/App/App/Info.plist:6]: Revert this file

File: src/main.ts [hunk 5]
```diff
@@ -18,7 +18,7 @@ Sentry.init({
     }),
```
Comment (yksolanki9) [inline src/main.ts:18]: Revert this change

**Review by yksolanki9 — CHANGES_REQUESTED** [review]
Please remove changes in gradle and capacitor files

**Review by Julias0 — CHANGES_REQUESTED** [review]
Please send for me to review once @yksolanki9 's comments are taken care of and approved

File: src/main.ts (hunk 5 above)
Comment (yksolanki9) [reply src/main.ts:18]: This file is not reverted yet

**Review by yksolanki9 — CHANGES_REQUESTED** [review]

File: android/app/build.gradle [hunk 6]
```diff
@@ -6,7 +6,7 @@ android {
         applicationId "com.ionicframework.fyle595781"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 51000
+       versionCode 51000
```
Comment (yksolanki9) [inline android/app/build.gradle:9]: Remove the extra space

File: ios/App/App.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -398,7 +398,7 @@
 				INFOPLIST_FILE = App/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				MARKETING_VERSION = 5.10.0;
+				MARKETING_VERSION = 5.33.0;
```
Comment (yksolanki9) [inline ios/App/App.xcodeproj/project.pbxproj:401]: Revert this file. These are placeholder texts, neednt be updated

**Review by yksolanki9 — APPROVED** [review]
Left minor comments

**Review by Julias0 — APPROVED** [review]

**Pull request merged**